### ISSUE TITLE
New Update I am confused

### ISSUE THREAD

**Issue by rewop777** [AHAAAAAAA/PokemonGo-Map#2423]
First off i dont know which direction is right, download from the wiki, 1.21 i believe or use Git Bash to download it. 

Secondly I dont know which direction is correct, the wiki install instructions for windows that use the easy setup instuction or the github method

Thirdly i dont know if i should be running the $ Python command on CMD or on GIT HUB

**Comment by rewop777** [comment]
Also the search feels much slower, like MUUUUUUUUUCH slower than the last version i had which was when it was implemented with GIT Hub

**Comment by wwwescape** [comment]
It's extremely slow for me too. Previously, the map would fill up my screen in a couple of seconds. Now it takes more than 20-30 mins to fill up my screen. Not ideal if your hunting.

Also, any idea if Heroku deployments update automatically when commits are made to the repo?

**Comment by cyan** [comment]
well it seems that the time between each request is longer and not it 1 circle while before it was 3 circle by 3 circle if i remember well

**Comment by SethG911** [comment]
Lower the scan-delay back down to 1. The new default is set to 5.

**Comment by cyan** [comment]
yeah just tested that and it seems to be back to older version speed

**Comment by Simbe90** [comment]
Why did they increase the new default from 1 to 5? Is there any reason for that? I lowered the scan-delay back to 1 and put number of threads (-t) to 18, super fast scanning now, but are there any reason why I shouldn't run it like this? Greater risk of getting my scanning account banned?

**Comment by lisandro52** [comment]
@Simbe90 Check #1451 . It's all part of being an ethical web scraper.

**Comment by Simbe90** [comment]
Well that does make sense, at least for now as their servers are so bad.

**Comment by Encouraging** [comment]
How do i change these settings on Heroku

**Comment by Pringles91** [comment]
i downloaded the latest version from the dev, and in the config folder the config.ini is no longer called config.ini.example, BUT under the "type" section it is called an "example file" ive tried changing the scan delay and number of threads and NOTHING seems to be making this fast. it takes on average 7 seconds to scan a new thread.

**Comment by rewop777** [comment]
add the command to your line

-sd 1 

that should do it
